src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,95 @@
+import logo from "./logo.svg";
+
+import {
+  Container,
+  Grid,
+  Box,
+  Button,
+  Typography,
+  Link,
+  Avatar,
+} from "@mui/material";
+import { Facebook, GitHub, LinkedIn } from "@mui/icons-material";
+
+function Copyright() {
+  return (
+    <Typography variant="body2" color="text.secondary">
+      {"Copyright © "}
+      <Link color="inherit" href="https://softserveinc.com/">
+        SoftServe
+      </Link>{" "}
+      {new Date().getFullYear()}
+      {"."}
```

Comment:
what is `{" "}` `{"."}`

Reply:
{" "} - is for whitespace between SoftServe and 2022
{"."} - removed

Reply:
fixed, please check